lua/grpc-mission.lua

```diff
@@ -1,7 +1,18 @@
--- Set default settings.
 if not _G.GRPC then
   _G.GRPC = {}
 end
+
+-- load settings from `Saved Games/DCS/Config/dcs-grpc.lua`
+do
```

Comment:
Can we add some logging like.

* "Checking for DCS/Config/dcs-grpc.lua file"
* "DCS/Config/dcs-grpc.lua file not found"

To help with debugging?

Reply:
Done in 8cbf23ab6df327329a3a057aa87d5b61a5b44216 👍